content/lua/concepts/tables/tables.md

```diff
@@ -23,7 +23,21 @@ Table objects are created using a constructor expression, in this case, a pair o
 t = {}
 ```
 
```

Comment:
Add `## Syntax` section heading here.

Reply:
I don't understand sorry

Reply:
You just need to add the section heading. Please refer [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md?plain=1) line 19.

Reply:
Okii done